src/node/README.md

```diff
@@ -17,7 +19,21 @@ npm install grpc
 
 ## BUILD FROM SOURCE
  1. Clone [the grpc Git Repository](https://github.com/grpc/grpc).
- 3. Run `npm install`.
+ 2. Run `npm install` from the repository root.
+
+ - **Note:** On Windows, this might fail due to [issue# 4932](https:\github.com\nodejs\node\issues\4932) in which case, you will see something like the following in `npm install`'s output (towards the very beginning):
```

Comment:
I think you can just link to nodejs/node#4932, like in comments. In either case, it's worth saying that the bug is in Node, not our code.